frontend/src/api/fetchfriend.js

```
@@ -0,0 +1,40 @@
+    );
+    return response.data;
+  } catch (error) {
+    console.error(`Error fetching user details for ${userId}:`, error.message);
+    return { id: userId, username: `User ${userId}`, profile_picture: null };
+  }
+};
+
+export const fetchConnectedUsers = async () => {
+  try {
+    const token = localStorage.getItem("token");
+    const currentUserId = parseInt(localStorage.getItem("user_id"));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)